REVIEW.md

```diff
@@ -0,0 +1,85 @@
+- a new feature must be prefixed by `feat/` ex: `feat/new-contribution-form`
+- a bug fix can be prefiexed by `fix/`
+Or `chore`, `docs` or any other word you see fit!
+
+> We use Conventionnal commits for commit message, and this branch naming is inspired by the keywords use there
+
+## Commits
+For commit messages and we use Conventionnal commits convention.
+Beware, a single commit shouldn't break anything (nor the test, nor the app, nor anythign.
+We should be able to checkout any commits and run the app without any issues.
+
+> When unsure just squash everyhting.
+
+## History
+Keep your history clean! There can be many commits or only one, as long as the story told is consistent.
+If you the history doesn't feel right, just be a negationist and use (and abuse) `git rebase -i`. 
+Just as long as you don't reright other people's history it's fine (really).
+
+And if you feel like, feel freer to `squash` your branch history into a single commit.
+
+## Pull/Merge request
+When ready you can open a Pull Request targeting the main branch.
+> BTW shall we rename `master` to `main`?
+
+OR not! You can open the PR as soon as you create your branch, 
+this can be relevant to keep other team member posted. 
+Just be sure to add the WIP keyword somewhere in the title and/or add the `WIP` label to your PR.
+
+> Also if your PR relate to an issue in any way, don't forget to mention it (#42 for issue number 42)
+> https://www.conventionalcommits.org/en/v1.0.0-beta.2/#commit-message-for-a-fix-using-an-optional-issue-number
+
+> __Ultimate tip__ If your branch only contains one commit with a title and a body:
+> ```
+> doc: Add review document draft
+> At last, here's the long-awaited documentation!
+> ```
+> The title and description of the PR will use this information.
+
+## Reviewer - *the peer*
+Depending on the subject of you can assign non technical people as reviewers:
+- member of the product to review doc, story or screeshot
+- member of the comm team to review spelling, relevance of keyword use, ...
+- member of UX UI or graphical, to review screenshot and user stories in practice, 
+> to do this the best way possible feature branches could be deployed and testable soemwhere though
+
+- At least 2 reviewer should be assigned to a PR.
+- Reviewers!
+  - Please be dilligent with your reviews! From our experience, PR which aren't reviewed soon tend to wait... wait.. sometime for months...even years! (I just closed a 7 years old PR recently)
+  - Tip: If you don't understand anything you're reading, you're might not be the right person for the job! And that's OK! 
+  Just let us know and we'll help you understand or choose another person fitting the task (at least try to look for typo:))
+  - We love our work to be read by others (who said egoless?!) so please try to show you care and leave some comments in the diff!
+- A PR should be merged in the next 72h? at most
+- You must have at least one approval to merge a PR
```

Comment:
Paragraph proposal : 

## Assignment / Merge

We consider that the PR author is responsible for it during its lifecycle. It means that the author :
 - decides who to require a review from (but everyone can review/comment)
 - decides when he got sufficient feedbacks
 - decides whether to handle or not non-blocking comments
 - decides when to merge, and triggers it
 
 But then again, even if he has the final saying, we usually try to work a team, discuss things and find consensus.

Reply:
I like this, it's really helpful for the PR author to know that they are the one responsible for their PR.

Just so I understand, the PR author does *not* have the final say on merging something to main, correct? That responsibility should be with one of the reviewers right? I'm just thinking of a potential case where a PR author wants to merge but there are clear issues with the PR. In that case, it's best if someone with knowledge of the system makes the final call. What do you think?

Reply:
Discussed with @felix-lambert last week, and what he suggested kind of make sense:
- if the reviewer approved the PR, he should be the one to merge it

> Note: it avoids one extra cycle whereas the PR is approved and can be merged

> Note 2: Actually we kind of did that already but not always, I'm not really sure of the unwritten rule behind this.

But what about when there are 2 reviewers or more then? Is one approval still sufficient?
Shall it be a rule? An exception?

Reply:
> But what about when there are 2 reviewers or more then

- That’s why I suggested that the PR author is the one in position to know if he got enough feedback from review(s) or not, and decide to merge. Even if I get how it adds a «cycle» …

- It also solves the problem of considering if the remaining discussions are blocking or not

Of course, this only works assuming that PR author is experienced enough to be able make this decision.

In fact, we could adopt my proposal if the PR author is the maintainer, 
But if the author is not a maintainer, different rule : Any maintainer can merge.